packages/terra-form/src/components/Input.scss

```diff
@@ -0,0 +1,18 @@
+@import '~terra-mixins';
+@import '../_variables.scss';
```

Comment:
Fixed it here - https://github.com/cerner/terra-core/pull/352/commits/594716abdc84473dfcf3ced97dc9fb5897d5666c